src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -106,6 +106,8 @@
           url: /foundations/icons/octovisuals
         - title: Design guidelines
           url: /foundations/icons/design-guidelines
+    - title: System arguments
```

Comment:
I would really like to keep everything in one place if we can.

Reply:
Hey - apologies was out last week. I just noticed this merged 😬. I think it's a bit confusing to have the Rails development specific documentation in here, rather than in the repo. It's unclear from the nav (and title) that this is geared specifically towards Rails, whereas our other documentation that is framework-specific and living in the docs is clearly labeled as such (for ex - [React dev guidelines](https://primer.style/design/guides/development/react), [CSS utilities](https://primer.style/design/foundations/css-utilities/getting-started), and the React/Rails specific sections of components.